### ISSUE TITLE
Can you provide the complete training code, thank you.

### ISSUE THREAD

**Issue by iiecho1** [SanaNazari/LMFLoss#1]
Can you provide the complete training code, thank you.

**Comment by SanaNazari** [comment]
The experiments of the paper are done by transfer learning using ResNet50 and EfficientNetV2 pretrained on Imagenet. you can implement any code from other repos with these architectures and import the loss as your criterion.
